cinema/views.py

```diff
@@ -0,0 +1,35 @@
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+from rest_framework.generics import get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+
+@api_view(["GET", "POST"])
+def movies(request) -> Response:
+    if request.method == "GET":
+        movies_all = Movie.objects.all()
+        serializer = MovieSerializer(movies_all, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
```

Comment:
```suggestion
        serializer.is_valid(raise_exception=True):
        serializer.save()
        return Response(serializer.data, status=status.HTTP_201_CREATED)
```
since in case of invalid data, exception will be rised.